docs/modules/ROOT/pages/documentation/microprofile/config/dynamo-db.adoc

```diff
@@ -40,6 +40,8 @@ NOTE: The region name is the snake-case notation, not the full words. E.g. `eu-w
 
 ### From the Admin Console
 
+To configure the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` password aliases from the admin console, refer to the xref:documentation/payara-server/password-aliases/password-alias-admin-console-commands.adoc[Password Aliases Admin Console configuration] guide.
```

Comment:
Please move this page about Dynamo DB into the cloud section. It also uses `get-config-property --source cloud` so it should be in the same section as the orther cloud config sources.